open-metadata-publication/website/open-metadata-types/0598-Lineage-Relationships.md

```diff
@@ -0,0 +1,24 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
```

Comment:
These relationships really belong in Area 2 - they describe the linkage between assets and the linkages to other attachments.   All similar definitions are in Area 2.

Reply:
They were already defined (from v1.2 onwards) as part of 0598...  Though I guess the area is really just down to the documentation (nothing inherent in the types themselves), so I could change this in the diagrams and markdowns (?)

Reply:
(Or sorry, did you mean only the new ProcessHierarchy relationship should be in Area 2?)

Reply:
Have pushed a new commit with the entire set of relationships moved to `0298` instead of `0598` + linked these new markdowns and diagrams from the Area-2 base markdown.